site/docs/pages/index.mdx

```diff
@@ -223,12 +313,20 @@ setFilteredTokens(filteredTokens);
       </div>
     </div>
   </aside>
-  <aside className="md:px-2 pt-10">
-    <p className="text-lg md:text-xl py-4">
+
+
+  <main id="swap" className="pt-[56px] flex flex-col gap-4 md:flex-row md:text-left text-center">
```

Comment:
@abcrane123 We might want to start create components for all those sections... it's getting a lot of reduntant code.

Reply:
yeah agreed let me do that